pom.xml

```diff
@@ -223,18 +373,118 @@
 						</execution>
 					</executions>
 				</plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>xml-maven-plugin</artifactId>
+					<version>${xml-maven-plugin-version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.jboss.tattletale</groupId>
```

Comment:
IIRC you removed that dependency from the other PR #493

Reply:
I have update PR for the above comments, excepting the javadoc.